modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionsExchangeFuture.java

```diff
@@ -346,7 +346,7 @@
     private final GridDhtPartitionsStateValidator validator;
 
     /** Register caches future. Initialized on exchange init. Must be waited on exchange end. */
-    private IgniteInternalFuture<?> registerCachesFuture;
+    private IgniteInternalFuture<?> registerCachesFut;
```

Comment:
Such renaming just increases changes amount without real profit

Reply:
this is about abbr fix: Future -> Fut to decrease amount of warnings. I rely on this object, so picked the moment to fix this. Is there a process how to provide such fixes right way?

Reply:
Such fixes snatch code ownership (git annotations), so they should be performed only in case you really changing something.

Reply:
Got it. Fixed with re-push to keep the ownership